tests/test_things.py

```diff
@@ -59,9 +59,12 @@ def test_search(self):
         with self.assertRaises(ValueError):
             things.search("To-Do\0Heading")
 
-        # Find "To-Do in Heading" but not "Completed To-Do in Heading".
-        tasks = things.search("To-Do % Heading")
-        self.assertEqual(1, len(tasks))
+        todos = things.search("To-Do % Heading", status=None)
```

Comment:
I'd suggest that `status=None` is set by default, as this is how the Things GUI would work.

Reply:
@AlexanderWillner When I do "Quick Find" ⌘F in Things, I only see the incomplete ones. Only if I click on "Continue Search" do I see the other ones. What do you think? What makes most sense in practice as a user?

Reply:
Mh, good point. Then let’s keep it like this and see if there are change requests after #40